tests/test_routes.py

```diff
@@ -69,24 +113,24 @@ def test_download_post_unknown_format_from_api(mock_get_response, flask_test_cli
     assert response.status_code == 500
 
 
-@pytest.mark.usefixtures("mock_get_response_xlsx")
-def test_download_fails_csrf(flask_test_client):
-    flask_test_client.application.config["WTF_CSRF_ENABLED"] = True
-    response = flask_test_client.post("/download", data={"file_format": "json"})
-    assert response.status_code == 302
+# @pytest.mark.usefixtures("mock_get_response_xlsx")
+# def test_download_fails_csrf(flask_test_client):
+#     flask_test_client.application.config["WTF_CSRF_ENABLED"] = True
+#     response = flask_test_client.post("/download", data={"file_format": "json"})
+#     assert response.status_code == 302
 
 
-@pytest.mark.usefixtures("mock_get_response_xlsx")
-def test_download_filename_date(flask_test_client):
-    response = flask_test_client.post("/download", data={"file_format": "xlsx"})
+# @pytest.mark.usefixtures("mock_get_response_xlsx")
+# def test_download_filename_date(flask_test_client):
+#     response = flask_test_client.post("/download", data={"file_format": "xlsx"})
 
-    # Regex pattern for datetime format %Y-%m-%d-%H%M%S
-    datetime_pattern = r"^\d{4}-\d{2}-\d{2}-\d{6}$"
-    extracted_datetime = re.search(r"\d{4}-\d{2}-\d{2}-\d{6}", response.headers["Content-Disposition"]).group()
+#     # Regex pattern for datetime format %Y-%m-%d-%H%M%S
+#     datetime_pattern = r"^\d{4}-\d{2}-\d{2}-\d{6}$"
+#     extracted_datetime = re.search(r"\d{4}-\d{2}-\d{2}-\d{6}", response.headers["Content-Disposition"]).group()
 
-    # Assert datetime stamp on file is in correct format
-    assert re.match(datetime_pattern, extracted_datetime)
-    assert datetime.strptime(extracted_datetime, "%Y-%m-%d-%H%M%S")
+#     # Assert datetime stamp on file is in correct format
+#     assert re.match(datetime_pattern, extracted_datetime)
+#     assert datetime.strptime(extracted_datetime, "%Y-%m-%d-%H%M%S")
```

Comment:
If we no longer want these tests, let's just delete them rather than comment them out please.

Reply:
Deleted these tests.